api/preprint_providers/serializers.py

```diff
@@ -13,21 +13,38 @@ class PreprintProviderSerializer(JSONAPISerializer):
     ])
 
     name = ser.CharField(required=True)
-    description = ser.CharField(required=False)
+    description = ser.CharField(required=False, allow_null=True)
```

Comment:
Once these are not nullable at the model level it should probably change here too.